praw/exceptions.py

```diff
@@ -41,3 +41,16 @@ def __init__(self, error_type, message, field):
 
 class ClientException(PRAWException):
     """Indicate exceptions that don't involve interaction with Reddit's API."""
+
+
+class WebSocketException(ClientException):
+    """Indicate exceptions caused by use of WebSockets."""
+
+    def __init__(self, message, exception):
+        """Initialize a WebSocketException.
+
+        :param message: The exception message.
+        :param exception: The exception thrown by the websocket library.
+        """
+        super().__init__(message)
+        self.ws_exception = exception
```

Comment:
Can we be consistent with `prawcore` here, and call this `original_exception` please? https://github.com/praw-dev/prawcore/blob/master/prawcore/exceptions.py